src/button/buttonview.js

```diff
@@ -18,8 +21,87 @@ export default class ButtonView extends View {
 	/**
 	 * @inheritDoc
 	 */
-	constructor() {
-		super();
+	constructor( locale ) {
```

Comment:
Yea, but it means that you create a view with empty properties and then add properties to it. It means that button is in the invalid state for a moment. If we would add assertions to check if the view is not in the invalid state it would throw in this case. And, even if it is fine for the button to have no label nor image, it might be not fine for other views to have an undefined initial state.